src/hb-subset-plan.cc

```diff
@@ -269,6 +270,13 @@ _populate_gids_to_retain (hb_subset_plan_t* plan,
   }
 
   _remove_invalid_gids (plan->_glyphset, plan->source->get_num_glyphs ());
+  /* COLRv1 closure might introduce virtual glyph id which is
+   * greater than or equal to the numGlyph value in the 'maxp' table */
```

Comment:
Where did you get this from? It's not expected to.

Reply:
In this spec: https://github.com/googlefonts/colr-gradients-spec/blob/main/OFF_AMD2_WD.md
5.7.11.2.5.7 Format 11: PaintColrGlyph
It says PaintColrGlyph table might have a virtual glyph ID. So I thought we might include these virtual glyph IDs during closure.

Reply:
You are right. But we don't want to implement that. I'm pushing back about that part of the spec. Is insane.